src/index.html

```diff
@@ -2,29 +2,30 @@
 <html lang="en">
 
 <head>
-    <meta charset="utf-8">
-    <title>BonspielTrackingSystem</title>
-    <base href="/">
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <link rel="icon" type="image/x-icon" href="favicon.ico">
-    <link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500&display=swap" rel="stylesheet">
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <meta charset="utf-8">
```

Comment:
I'm not sure if these prettified files are necessary in this pull request.

Reply:
Right, thought it'll keep things standard... do let me know if you want my settings.json which uses this prettified settings. You can ignore the change on all the files

Reply:
Cool, I'll slip in my settings.json... it's included under .vscode/settings.json